src/panels/lovelace/cards/hui-history-graph-card.js

```diff
@@ -51,11 +57,18 @@ class HuiHistoryGraphCard extends PolymerElement {
   }
 
   setConfig(config) {
-    if (!config.entities || !Array.isArray(config.entities)) {
-      throw new Error('Error in card configuration.');
-    }
+    const entities = processConfigEntities(config.entities);
 
     this._config = config;
+
+    this.entities = [];
```

Comment:
Only assign it after you have collected all entities otherwise Polymer won't know the array has changed and the changed value will not propagate.